client/src/app/site/motions/modules/motion-workflow/components/workflow-list/workflow-list.component.ts

```diff
@@ -96,8 +96,7 @@ export class WorkflowListComponent extends BaseListViewComponent<ViewMotionWorkf
 
         dialogRef.afterClosed().subscribe(result => {
             if (result) {
-                throw new Error('TODO');
-                // this.workflowRepo.create(new MotionWorkflow({ name: result })).then(() => {}, this.raiseError);
+                this.workflowRepo.create(new MotionWorkflow({ name: result })).catch(this.raiseError);
```

Comment:
`catch` should not be necessary, since it is in the base repo, right? Same below.